core/contracts/common/test/WithdrawableTest.sol

```diff
@@ -17,7 +17,7 @@ contract WithdrawableTest is Withdrawable {
         require(msg.value > 0);
     }
 
-    function setInternalWithdrawRole(uint256 roleId) public {
-        _setWithdrawRole(roleId);
+    function setInternalWithdrawRole(uint256 _roleId) public {
```

Comment:
This appeases the `state-variable-shadowing` detector, because `WithdrawableTest` inherits `Withdrawable` and we should not have a function input named the same as Withdrawable's state variable `roleId`

Reply:
can we not use underscores to remain consistent with our styling rules? could you call this `setRoleId` or something different?

Reply:
ah duh good catch I'll make this change

Reply:
done